multiview_mapping_toolkit/constants.py

```diff
@@ -110,8 +110,8 @@ def hex_to_rgb(value):
 MATPLOTLIB_PALLETE = [
     hex_to_rgb(x) for x in plt.rcParams["axes.prop_cycle"].by_key()["color"]
 ]
-TEN_CLASS_VIS_KWARGS = {"cmap": "tab10", "clim": (-0.5, 9.5)}
-TWENTY_CLASS_VIS_KWARGS = {"cmap": "tab20", "clim": (-0.5, 19.5)}
+TEN_CLASS_VIS_KWARGS = {"cmap": "tab10"}
```

Comment:
If you choose to go this route, you'll need to update this code that expects `clim`: [here](https://github.com/open-forest-observatory/multiview-mapping-toolkit/blob/e674b880513b10efcc2e10aea0873ac9da57b134/multiview_mapping_toolkit/utils/visualization.py#L38). It's also optionally used [here](https://github.com/open-forest-observatory/multiview-mapping-toolkit/blob/e674b880513b10efcc2e10aea0873ac9da57b134/multiview_mapping_toolkit/utils/prediction_metrics.py#L45). In the later case, I'd switch it to a more conventional `vmin/vmax` if you're not using `clim` anywhere.

Reply:
For now I've added them back in, since the code in meshes.py relies on the get_IDS_to_labels() function which we can't use in the other code.